boost_python is found for the correct python version in the Makefile.
fix
> Could not find a package configuration file provided by "boost_python3"

--- cv_bridge/CMakeLists.txt~	2022-10-03 12:08:47.000000000 +0200
+++ cv_bridge/CMakeLists.txt	2024-06-19 17:08:02.847640307 +0200
@@ -6,13 +6,8 @@
 if(NOT ANDROID)
   find_package(PythonLibs)
 
-  if(PYTHONLIBS_VERSION_STRING VERSION_LESS "3.8")
-    # Debian Buster
-    find_package(Boost REQUIRED python37)
-  else()
     # Ubuntu Focal
     find_package(Boost REQUIRED python)
-  endif()
 else()
 find_package(Boost REQUIRED)
 endif()
